scripts/partition_manager/partition_manager.rst

```diff
@@ -79,6 +102,7 @@ Each partition is defined as follows:
          property_value
 
 *partition_name* is the name of the partition (for example, ``mcuboot``).
+
```

Comment:
Intentional?

Reply:
Yes - the next sentence is not related, so we need to start a new paragraph.